package-lock.json

```diff
@@ -0,0 +1,671 @@
+{
```

Comment:
Oops! This file shouldn't be committed.

Reply:
We should add it to .gitignore. Feel free to do that here if you want.

Reply:
ah understood, I've added updated the gitignore